source/monsters/monsters_a-z/index.rst

```diff
@@ -1,12 +1,16 @@
 
 .. _srd:monsters-a-z:
 
-Monsters A-Z
+Monster Descriptions
 -------------------------------------------------------------
 
-.. toctree::
-   :name: monsters-az
-   :titlesonly:
-   :glob:
-   
-   */index
+.. container:: columns
+
+    .. toctree::
+      :name: monsterlist
+      :titlesonly:
+      :glob:
+
+      */index
+      Miscellaneous Creatures <../miscellaneous-creatures/index>
+      Nonplayer Characters <../nonplayer-characters/index>
```

Comment:
I agree that we shouldn't break outside links unnecessarily, but I also don't want that concern to become something that drags us down. If it's a strong concern we should figure out an automated way of addressing it, e.g. a better 404 page, auto-created redirects, etc. Part of the whole reason we chose Sphinx was that it allowed us to move things without breaking the site.